rest/rest_service.py

```diff
@@ -142,6 +143,7 @@ def __init__(self, settings_name):
         self.wrapper = SettingsWrapper()
         self.logger = None
         self.app = Flask(__name__)
+        CORS(self.app)
```

Comment:
the 'No 'Access-Control-Allow-Origin'' should be not be an issue as the CORS modules adds the required HTTP headers to the HTTP messages to allow cross origin requests. 

Following your second EDIT .... are you still getting that error?